Sources/OpenCombine/Published.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Published.swift
+//  OpenCombine
+//
+//  Created by Евгений Богомолов on 01/09/2019.
+//
+
+#if swift(>=5.1)
+@propertyWrapper public struct Published<Value> {
```

Comment:
Please add documentation for the struct:

```swift
/// Adds a `Publisher` to a property.
///
/// Properties annotated with `@Published` contain both the stored value
/// and a publisher which sends any new values after the property value
/// has been sent. New subscribers will receive the current value
/// of the property first.
```

Reply:
Done